src/sentry/static/sentry/app/views/eventsV2/eventDetails/pagination.tsx

```diff
@@ -62,74 +62,40 @@ const Pagination = (props: Props) => {
   const links = buildTargets(event, eventView, organization);
 
   return (
-    <Paginator>
-      <StyledIconLink
+    <Paginator merged>
+      <Button
+        size="small"
         to={links.oldest || ''}
         disabled={links.previous === null || links.oldest === null}
-      >
-        <InlineSvg src="icon-prev" />
-      </StyledIconLink>
-      <StyledTextLink
+        icon={<IconPrevious size="xs" />}
```

Comment:
@markstory so the icons are only broken in `app/components/pagination`? 👀

Reply:
No the chevron icons don't render in percy anywhere. The same icon is used in the sidebar for the collapse sidebar item. That sidebar item has no icons in any percy diff.

Reply:
Ah the chevron -- I just checked Percy and there seems to be a firefox upgrade, I'll see if the sidebar icon shows up in future builds or not.

@doralchan any ideas why this icon wouldn't be rendering? I don't know what version of firefox we currently use on percy (but mark checked his local firefox and it renders fine).